src/list.tsx

```diff
@@ -724,13 +728,14 @@ type BoxedRowProps = ExclusifyUnion<
 >;
 
 export const BoxedRow = React.forwardRef<HTMLDivElement, BoxedRowProps>(({dataAttributes, ...props}, ref) => (
-    <Boxed
-        isInverse={props.isInverse}
+    <InternalBoxed
+        isInverse={props.isInverse && !props.danger}
+        background={props.danger && props.isInverse ? vars.colors.backgroundContainerError : undefined}
```

Comment:
this is weird, in my opinion, but I'm just following the specs

Reply:
removed this case, after talking with design. `<BoxedRow isInverse danger/>` is not an allowed case